CODEOWNERS

```diff
@@ -23,24 +23,24 @@
 
 # These files change frequently and changes are high-risk
 
-/src/cbmc/ @smowton @kroening @tautschnig @peterschrammel
-/src/goto-programs/ @smowton @kroening @tautschnig @peterschrammel @pkesseli
-/src/util/ @smowton @kroening @tautschnig @peterschrammel @pkesseli
-/src/solvers/refinement @martin-cs @romainbrenguier @peterschrammel
-/jbmc/src/java_bytecode/ @smowton @mgudemann @thk123 @cristina-david @cesaro @pkesseli @NathanJPhillips @peterschrammel
-/src/analyses/ @martin-cs @peterschrammel @chrisr-diffblue @thk123 @smowton
-/src/pointer-analysis/ @martin-cs @peterschrammel @chrisr-diffblue @smowton
+/src/cbmc/ @kroening @tautschnig
+/src/goto-programs/ @kroening @tautschnig
+/src/util/ @kroening @tautschnig
+/src/solvers/refinement @romainbrenguier @peterschrammel
+/jbmc/src/java_bytecode/ @peterschrammel @smowton
```

Comment:
I think this isn't workable without someone from the Deeptest group as a code owner for `java_bytecode` as it needs to be changed frequently.